src/models/pang15/repairs.md

```diff
@@ -0,0 +1,242 @@
+# Pangolin (Parts & Repairs)
+
+Many components in your Pangolin can be upgraded or replaced as necessary. Follow these step-by-step guides for instructions:
+
+- [Removing the bottom cover](#removing-the-bottom-cover)
+- [Replacing the RAM](#replacing-the-ram)
+- [Replacing an M.2/NVMe SSD](#replacing-an-m2nvme-ssd)
+- [Removing the battery](#removing-the-battery)
+- [Reseating the WiFi/Bluetooth antennas](#reseating-the-wireless-antenna-connectors)
+- [Replacing the CMOS battery](#replacing-the-cmos-battery)
+- [Replacing the fans/heatsink/thermal paste](#replacing-the-cooling-system)
+- [Replacing the speakers](#replacing-the-speakers)
+
+## Removing the bottom cover:
+
+Removing the cover is required to access the internal components. Prior to removing the cover, ensure the AC power is unplugged and all peripherals (including SD cards and USB drives) are unplugged or removed from the system.
+
+**Part numbers:**
+- The bottom panel's part number is `02.SKD.NM16AHFTX02`.
+
+**Tools required:** Cross-head (Phillips) screwdriver  
+**Time estimate:** 5 minutes  
+**Difficulty:** Easy <span style="color:green;">●</span>  
+
+### Steps to remove the bottom cover:
+
+1. Place the machine lid-side down.
+    - Use a soft surface (such as a towel) to avoid scratches.
+2. Remove the 11 bottom panel screws.
+    - The back-center screw (highlighted red below) is longer than the rest of the screws.
+    - The front four screws are mounted at an angle into the machine; hold the screwdriver at an angle for the easiest removal/installation.
+
+![Bottom panel screws](./img/bottom-panel-screws.webp)
+
+3. Lift the bottom panel off, starting from the hinges in the back.
+
+## Replacing the RAM:
+
+The Pangolin 15 supports up to 96GB (2x48GB) of DDR5 SO-DIMMs running at 6400MHz. If you've purchased new RAM, need to replace your RAM, or are reseating your RAM, follow these steps.
```

Comment:
Similar note about the tech specs on the website and the photo has 4800 Mhz installed.

Reply:
Corrected speed to 5600 in both locations.